src/main/java/com/treasuredata/client/TDClient.java

```diff
@@ -993,4 +1010,16 @@ else if (!jobRequest.getResultOutput().isEmpty()) {
             return Optional.absent();
         }
     }
+
+    @Override
+    public TDImportResult importFile(String databaseName, String tableName, String format, File file)
+    {
+        return doPut(buildUrl(String.format("/v3/table/import/%s/%s/%s", databaseName, tableName, format)), ImmutableMap.of(), file, TDImportResult.class);
+    }
+
+    @Override
+    public TDImportResult importFile(String databaseName, String tableName, String format, File file, String id)
```

Comment:
I think we also need to pass msgpack.gz format option.

Reply:
See https://github.com/treasure-data/td-api/blob/master/app/controllers/api_v3/api_imports_controller.rb#L13 (private repo)

Reply:
'table/import_with_id/:database_name/:table_name/:unique_id/:data_format'

Reply:
So it means the commend should describe that the data file must be in msgpack.gz format.

Reply:
ok, we can hard-code it